### DISCUSSION TITLE
"Key Count" mod to solve inconsistency caused by mania's 10K mod

### DISCUSSION THREAD

**Discussion by Chiffario** [ppy/osu#23096] [Ideas]
Currently almost all mods (for both lazer and stable) have 2-letter acronyms. This behaviour is basically consistent and, due to it's consistency, it allows to work with mods in easy ways. However, there's one major exception - osu!mania's 10K mod. It is the only mod in the game that has a 3 letter acronym, as shown by the [osu-web mods.json](https://github.com/ppy/osu-web/blob/master/database/mods.json#L2470). Having an exception like this doesn't sound good from the maintenance standpoint.

Now, I don't know if this would be a good or even a plausible solution, but shifting all the key conversion mods into a single mod with multiple settings would remove the inconsistency

**Comment by peppy** [comment]
We're going to run out of two letter acronyms anyway, with the number of new mods that are being made. I don't see this as an issue.
